### PR TITLE
Update BSci script

### PR DESCRIPTION
**Pull request by danlessa**
##### Description

1. Refactored the `bsci_script` method for correctly importing the squelches from the BSci pipeline
2. Added a `ToggleUser` class and `FlaggingScriptOutput` type for describing the `bsci_script` output
3. Refactored `process_bsci_sybil_csv` so that toggling the sybil users happen outside the `bsci_script`

### CODE REVIEW

File: app/grants/utils.py [hunk 1]
```diff
@@ -392,88 +395,73 @@ def toggle_user_sybil(sybil_users, non_sybil_users):
...
+        df.loc[labels_by_prediction, 'is_sybil'] = df[labels_by_prediction].prediction_score
+        df.loc[labels_by_prediction, 'label'] = "ML Prediction"
+        
+        # Generate dict records
+        sybil_records = df.query('is_sybil == True').to_dict('records')
+        non_sybil_records = df.query('is_sybil == False').to_dict('records')
+        
+        sybil_records = [ToggleUser(**d) for d in sybil_records]
+        non_sybil_records = [ToggleUser(**d) for d in non_sybil_records]
+
```
Comment (gdixon) [inline app/grants/utils.py:458]: Could we add back the call to `toggle_user_sybil(sybil_users, non_sybil_users)`:

```suggestion
                toggle_user_sybil(sybil_records, non_sybil_records)
                
```

Comment (gdixon) [reply app/grants/utils.py:458]: Or alter [app/grants/tasks.py -> process_bsci_sybil_csv()](https://github.com/gitcoinco/web/blob/5db13f06b7b3dedaa683f504f045006d7fe95b8a/app/grants/tasks.py#L443) to make this call with the return data.

**Comment by danlessa** [comment]
I've updated so that `tasks.py` is calling the `toggle_user_sybil` with the `bsci_script` output

File: app/grants/tasks.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from django.conf import settings
 from django.utils import timezone
 from django.utils.text import slugify
+from app.grants.utils import toggle_user_sybil
```
Comment (gdixon) [inline app/grants/tasks.py:10]: Can we combine this with the other `grant.utils` import? (on ln 17/18)

**Comment by thelostone-mc** [comment]
- I'm not very clear on what this change here is. Could we update the description ?
- Was this tested locally ? If so 
  - did the S3 upload work as expected
  - do the users marke sybil -> actually get marked sybil ?
  - do the users marked non sybil -> has the sybil mark removed ?

**Review by thelostone-mc — COMMENTED** [review]
- I'm not very clear on what this change here is. Could we update the description ?
- Was this tested locally ? If so 
  - did the S3 upload work as expected
  - do the users marke sybil -> actually get marked sybil ?
  - do the users marked non sybil -> has the sybil mark removed ?

**Comment by danlessa** [comment]
> * I'm not very clear on what this change here is. Could we update the description ?

Done!

> * Was this tested locally ? If so
>   
>   * did the S3 upload work as expected
>   * do the users marke sybil -> actually get marked sybil ?
>   * do the users marked non sybil -> has the sybil mark removed ?

I do not have the proper environment and credentials to validate those items. I've done some restricted tests and it seems to be working ok

**Pull request merged**